3574/2/commands/command_checkin.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../helpers/session_state'
+require_relative '../helpers/location'
+require_relative '../helpers/photo'
+require_relative '../helpers/save'
+
+module CheckinCommand
+  include SessionStatus
```

Comment:
you shouldn't have included modules inside your module. try to find different opportunities to use methods from other modules